### PR TITLE
SA342 Gazelle: Copilot support.

### PR DESCRIPTION
**Pull request by gcask**

### CODE REVIEW

File: Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua [hunk 1]
```diff
@@ -1619,105 +1619,124 @@ function SR.exportRadioUH1H(_data)
...
+        uhfPush = 457 -- IC2_UHF_Push
+    end
     
-        _data.radios[3].name = "UHF TRA 6031"
 
-        -- deal with odd radio tune & rounding issue... BUG you cannot set frequency 243.000 ever again
-        local freq = SR.getRadioFrequency(31, 500)
-        freq = (math.floor(freq / 1000) * 1000)
+    _data.radios[2].name = "TRAP 138A"
+    local MHZ = 1000000
```
Comment (ciribob) [inline Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua:1653]: How come you've switch the methods for getting the radio frequency? It should work if you're in the pilot or copilot seat (or the gunner)?

Or are they not syncing?

File: Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua [hunk 2]
```diff
@@ -1619,105 +1619,124 @@ function SR.exportRadioUH1H(_data)
...
+    local fm1Push = 453 -- IC1_FM1_Push
+    local uhfPush = 454 -- IC1_UHF_Push
+
+    if _seat == 1 then
+        -- Copilot.
+        vhfVolume = 79 -- IC2_VHF
+        uhfVolume = 80 -- IC2_UHF
+        fm1Volume = 81 -- IC2_FM1
+
+        vhfPush = 455 -- IC2_VHF_Push
```
Comment (ciribob) [inline Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua:1646]: Have they added animations to these again? They did remove the clickability of these a while ago (for some reason)

Comment (gcask) [reply Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua:1646]: I think? They are in the arg_int.lua and I was able to click around and play with the volume for the copilot's knobs...

File: Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua (hunk 1 above)
Comment (gcask) [reply Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua:1653]: I have done no such thing? The TRAP is still radio 2, and the UHF radio 3...

I don't know what's up with the diff here. If you open the file for this PR and [go to the relevant line](https://github.com/ciribob/DCS-SimpleRadioStandalone/blob/8a82a831a8c29d546ecca146803a67ea5e621e5d/Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua#L1652) it should look correct.

File: Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua (hunk 2 above)
Comment (gcask) [reply Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua:1646]: They're also in the clickabledata.lua, circa line 325.

Comment (ciribob) [reply Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua:1646]: Ah great!

File: Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua (hunk 1 above)
Comment (ciribob) [reply Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua:1653]: Oh nice! Sorry now it makes sense 👍

File: Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua [hunk 3]
```diff
@@ -1619,105 +1619,124 @@ function SR.exportRadioUH1H(_data)
...
         _data.control = 1; -- COCKPIT Controls
-    
     else
-        _data.control = 0; -- no copilot  radio controls - allow them to switch
-        
-        _data.radios[1].volMode = 1 
-        _data.radios[2].volMode = 1 
-        _data.radios[3].volMode = 1 
+        -- Neither Pilot nor copilot - everything overlay.
+        _data.capabilities.dcsRadioSwitch = false
```
Comment (ciribob) [inline Scripts/DCS-SRS/Scripts/DCS-SimpleRadioStandalone.lua:1729]: They also lose hotmic (unfortunately)

Cant have hotmic and non cockpit controls

**Comment by ciribob** [comment]
Thank you for this!

**Pull request merged**